### DISCUSSION TITLE
Image shows different width on different platforms

### DISCUSSION THREAD

**Discussion by KJ-Chiu** [dolanmiu/docx#867] [Q&A] [answered]
## Problem I faced
The images in table display in quite different width between Windows and MacOS
> also different in Google docs but I had already given up lol

I had tried inline and floating position but still can't fixed this question. Does anyone know the correct setting or face the same problems?

### In Windows
![image](https://user-images.githubusercontent.com/40239349/113173965-82068f00-927c-11eb-8165-cf4adf8d976c.png)

### In MacOS (photo by colleague, about 1/2 width and height only)
![image](https://user-images.githubusercontent.com/40239349/113174036-921e6e80-927c-11eb-8c6c-38b57daff4e2.png)

**Comment by dolanmiu** [comment]
Could you show me the code which created this?

**Comment by KJ-Chiu** [comment, marked as answer]
After several hours debugging I found the error came from another npm package - html-to-image.
Different size of screen get different blob result though they look same when download it.

I turn to ask server side render specific width and height image instead of doing this at frontend then everything goes right.
> except google docs but never mind

- here is the sample code I used before
https://codesandbox.io/s/generate-docx-with-player-list-table-qpt96

Still, thanks for helping.

**Reply by dolanmiu** [reply]
No problem!
